src/concrete/ml/onnx/convert.py

```diff
@@ -208,7 +221,13 @@ def get_equivalent_numpy_forward_from_onnx(
             f"Available ONNX operators: {', '.join(sorted(IMPLEMENTED_ONNX_OPS))}"
         )
 
+    # Compute the LSB to be remove manually while waiting for the truncate feature
+    # FIXME: https://github.com/zama-ai/concrete-ml/issues/397
```

Comment:
`https://github.com/zama-ai/concrete-ml/issues/397` still works but I think it's better if you change it to `https://github.com/zama-ai/concrete-ml-internal/issues/4143`